src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -16,7 +16,12 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
         {
             Capabilities = new ServerCapabilities
             {
-                // Placeholder until more capabilities are enabled
+                CompletionProvider = new CompletionOptions()
+                {
+                    AllCommitCharacters = new[] { " ", ".", ";", ">", "=" }, // This is necessary to workaround a bug where the commit character in CompletionItem is not respected.
```

Comment:
Did you happen to file an issue for this?

Reply:
Going to start an email thread about this.